Content.Server/Chemistry/Metabolism/DamageMetabolism.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Server.GameObjects.Components.Nutrition;
+using Content.Shared.Chemistry;
+using Content.Shared.Interfaces.Chemistry;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Shared.Damage;
+using Content.Shared.GameObjects.Components.Damage;
+
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Default metabolism for medicine reagents. Attempts to find a DamageableComponent on the target,
+    /// and to update its damage values.
+    /// </summary>
+    [DataDefinition]
+    public class DamageMetabolism : IMetabolizable
```

Comment:
since this also handles healing chems it should probably be called HealthChangeMetabolism or something similar

Reply:
I'm not so sure, because DamageMetabolism is less clunky, and tells the samething effectively, but other people might have something to say on it

Reply:
"DamageMetabolism" to me implies that you're damaging an entity, not healing them

Reply:
Ok, fair point should be changed

Reply:
I mean technically healing is just reverse damage and it could function as both.